web/auth.go

```diff
@@ -115,3 +117,8 @@ func (h *handler) clearSession(w http.ResponseWriter, r *http.Request) {
 
 	http.Redirect(w, r, h.config.Base, http.StatusTemporaryRedirect)
 }
+
+func getSessionStorageKey(username string, password string) []byte {
```

Comment:
nit: Call this `generateSessionStorateKey`

Reply:
done :)

Reply:
dmn, you misstype))